assignments/arrays.js

```diff
@@ -65,30 +64,65 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+console.log("Car 33 is a ", inventory[33].car_year, inventory[33].car_make, inventory[33].car_model);
```

Comment:
remember inventory[33] is looking for the item at index 33. the car with the ID 33 is actually index 32